Slim/Web/Template/SkinManager.pm

```diff
@@ -263,9 +263,9 @@ sub _parseURIs {
 
 	return $text unless $text;
 
-	if (!($text =~ s!\b(https?://[A-Za-z0-9\-_\.\!~*'();/?:@%&=+$,]+)!<a href=\"$1\" target=\"_blank\" class="link">$1</a>!igo)) {
+	if (!($text =~ s!\b(https?://[-A-Z\d_.~*'();/?:@%&=+,#\$\!]+)!<a href=\"$1\" target=\"_blank\" class="link">$1</a>!igo)) {
```

Comment:
Does moving the exclamation point have any significance?

Reply:
No, it was simply to group all the escaped items at the end of the RE.